### ISSUE TITLE
LIBS variable in Makefile not read correctly

### ISSUE THREAD

**Issue by jortbmd** [bmd-studio/stm32-for-vscode#91]
When adding USBPD to the library in the makefile it adds it below the LIBS variable:
``` Makefile
# libraries
LIBS = -lc -lm -lnosys  \
-l:USBPDCORE_PD3_FULL_CM0PLUS_wc32.a
```
When doing this STM32 for VSCode does not include the default libraries e.g. -lc -lm -lnosys

This results in the following error:
```
in function `log':
w_log.c:(.text.log+0x48): undefined reference to `__errno'
````

**Comment by jortbmd** [comment]
For anyone running into the same issue. A quick fix is to just add the libraries to the libaries section in the STM32-for-VSCode.config.yaml
e.g.:
``` Yaml
# libraries to be included. The -l prefix to the library will be automatically added.
# Mind that non standard libraries should have a path to their respective directory.
libraries: 
  - :USBPDCORE_PD3_FULL_CM0PLUS_wc32.a
  - c
  - m
  - nosys
```

**Comment by jortbmd** [comment]
Fixed in 3.2.3 release.
